### DISCUSSION TITLE
Can't create API keys

### DISCUSSION THREAD

**Discussion by seba9999** [ovh/manager#5944] [Q&A] [answered]
Hello I'm trying to play with the API but can't create my API keys ... 

This site https://eu.api.ovh.com/createToken/  give me an annoying "Invalid account / password". 

But I'm able to play around with the API with the web version because I can authentify at https://eu.api.ovh.com/auth/ 

What do I miss ?

Btw I've also "activated" the "developer options" in my OVH account (Which talks about API....) 

PS : trying to follow the "advanced" part of this [official tutorial](https://docs.ovh.com/gb/en/api/first-steps-with-ovh-api/#advanced-usage-pair-ovhcloud-apis-with-an-application_2)

**Comment by seseso** [comment]
I have exactly the same problem. Did you had any answer on that? Have you contacted support?

**Comment by seseso** [comment, marked as answer]
Well, the solution is simple:
Use your NIC HANDLE, without the suffix "-ovh" as the username. 

It worked for me

**Reply by edwinrolle** [reply]
Didn't work for me, I tried all usernames (NIC handle, email, username)

**Comment by mrabdibdi-anvil** [comment]
Hello,
if it helps someone 🤷‍♂️ .. 
I found the answer in the README of this repo: 

https://github.com/ovh/node-ovh
